pkg/authorization/api/types.go

```diff
@@ -160,7 +160,7 @@ type Policy struct {
 	LastModified util.Time
 
 	// Roles holds all the Roles held by this Policy, mapped by Role.Name
-	Roles map[string]Role
+	Roles map[string]*Role
```

Comment:
> so the external types are structs and the internal types are pointers?

Yes.   We cannot logically hold nils in the maps, so our external packages should help enforce that.